### PR TITLE
Add Penetration Testing

### PR DESCRIPTION
**Pull request by fsacer**
Added awesome-pentest again because https://github.com/sindresorhus/awesome/pull/841 was closed and this list seems to now conform to guidelines.

**[Ihttps://github.com/enaqx/awesome-pentest]**

**[a great resource for pentesting]**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Has the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
> Include a short description about the project/theme of the list. **It should not describe the list itself.**

:arrow_up:

---

The license badge should be SVG and placed at the bottom. See this readme for example.

**Comment by sindresorhus** [comment]
// @enaqx

**Comment by fsacer** [comment]
// @meitar @techgaun

**Comment by meitar** [comment]
I am not convinced we are ready; I have been going over `awesome-pentest` list slowly, see, enaqx/awesome-pentest#86 but we still, for instance, do not have `awesome` and `awesome-list` GitHub topics, along with a couple other minor things. I am working on it and I appreciate the ping, though.

**Comment by sindresorhus** [comment]
Ping

**Comment by techgaun** [comment]
@sindresorhus @fsacer I will spend some time this weekend to make the list compliant and ping back here. thanks!

**Comment by meitar** [comment]
Thanks for checking in on this, @techgaun.

**Comment by sindresorhus** [comment]
Ping

**Comment by techgaun** [comment]
@sindresorhus very sorry about not keeping this updated. I have been really busy due to our accelerator program at work and have more stuff for at least two more weeks. I tried contacting the repo owner to update the topics but haven't heard back from him yet. Hopefully, I will get something done in next two weeks & also hear from trepo owner.

**Comment by enaqx** [comment]
@techgaun Have busy days also. What actions required?

**Comment by techgaun** [comment]
@enaqx we need to add github topics: `awesome-list` & `awesome` at the very list. would be good to have more relevant topics too. Also, if possible, can you provide access to me to do so in the future? If not, all good. thanks!

**Comment by enaqx** [comment]
Not sure if GitHub provides admin rights for personal repos.

**Comment by davisonio** [comment]
> Not sure if GitHub provides admin rights for personal repos.

it does, you can go to Settings > Collaborators

**Comment by pokeball99** [comment]
@sindresorhus @fsacer is this repo ready yet to be added?

**Comment by meitar** [comment]
Thank you everyone for your good work in curating this list and conforming it to Awesome List guidelines. :)

File: readme.md [hunk 1]
```diff
@@ -407,6 +407,7 @@
 - [Incident Response](https://github.com/meirwah/awesome-incident-response)
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security) - Security of web apps & services.
+- [Penetration Testing](https://github.com/enaqx/awesome-pentest)
```
Comment (sindresorhus) [inline readme.md]: `https://github.com/enaqx/awesome-pentest` => `https://github.com/enaqx/awesome-pentest#readme`

**Comment by sindresorhus** [comment]
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.

This is not done.

**Comment by sindresorhus** [comment]
You also need to resolve the merge conflict.

**Comment by meitar** [comment]
@sindresorhus I have taken the liberty of going through your new contribution guidelines and producing PR #1366 intended to obsolete this one, as I cannot push to the WIP branch here.

**Comment by sindresorhus** [comment]
Closing in favor of #1366

**Pull request closed without merging**